src/main/java/com/kainos/ea/models/Job.java

```diff
@@ -0,0 +1,53 @@
+package com.kainos.ea.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.checkerframework.checker.signature.qual.Identifier;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+public class Job {
+
+    @NotEmpty
+    @Identifier
+    private int id;
+    @NotEmpty
+    private String name;
+    @NotEmpty
+    private String job_description;
+
+    @JsonCreator
+    public Job(@JsonProperty("jobId") @Identifier int id, @JsonProperty("jobName") String name, @JsonProperty("job_description") String job_description) {
```

Comment:
I think there is some unnecessary complexity here? If you keep your fields the same name, you won't need to use JsonCreator for the unmarshalling.

Reply:
We were following Shaun's practice in his example, so we would be unsure how to change this

Reply:
As discussed on the call, this can be updated - give it a go and come back to me with any questions :)